src/main/java/com/northconcepts/datapipeline/examples/shopify/ReadShopifyInventoryItemsWriteToJson.java

```diff
@@ -0,0 +1,31 @@
+package com.northconcepts.datapipeline.examples.shopify;
+
+import com.northconcepts.datapipeline.foundations.file.LocalFileSink;
+import com.northconcepts.datapipeline.foundations.pipeline.Pipeline;
+import com.northconcepts.datapipeline.foundations.pipeline.PipelineInput;
+import com.northconcepts.datapipeline.foundations.pipeline.PipelineOutput;
+import com.northconcepts.datapipeline.foundations.pipeline.input.DataReaderPipelineInput;
+import com.northconcepts.datapipeline.foundations.pipeline.output.JsonRecordPipelineOutput;
+import com.northconcepts.datapipeline.shopify.ShopifyInventoryItemReader;
+
+public class ReadShopifyInventoryItemsWriteToJson {
```

Comment:
No need to say everything in the class name: `ReadShopifyInventoryItems` is enough.  We'll create a too later that follows your thinking.

Please update the other examples similarly.

Reply:
done